### PR TITLE
Fix some typos in Figure Github workflow

### PR DESCRIPTION
**Pull request by truongnh1992**

### CODE REVIEW

File: contributors/guide/github-workflow.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ echo export PATH="\$PATH:$working_dir/kubernetes/third_party/etcd" >> ~/.profile
 ```sh
 cd $working_dir/kubernetes
 
-# Run all the presubmission verification. Then, run a specific update script (hack/update-*.sh)
+# Run all the pre-submission verification. Then, run a specific update script (hack/update-*.sh)
```
Comment (nikhita) [inline contributors/guide/github-workflow.md]: `presubmission` is correct: http://www.yourdictionary.com/presubmission.

**Comment by nikhita** [comment]
@truongnh1992 Can you mention what typos are being corrected in the diagram in the PR body?

File: contributors/guide/github-workflow.md (hunk 1 above)
Comment (truongnh1992) [reply contributors/guide/github-workflow.md]: Thank you. I've just reverted it.

**Comment by truongnh1992** [comment]
> 
> 
> @truongnh1992 Can you mention what typos are being corrected in the diagram in the PR body?

I modified typo in "After **Step 7**, if has...":
futher -> further

**Comment by truongnh1992** [comment]
> 
> 
> > @nikhita I modified typo in "After Step 7, if has...":
> > futher -> further
> 
> @truongnh1992 Thanks! Can you also mention this in the main PR body so that someone who looks at this PR can come to know what it does?
> 
> This lgtm, but will leave it to @castrojo or @cblecker to add the label.

I got it. Thank you.

**Pull request merged**